akka-docs/src/main/paradox/typed/persistence-snapshot.md

```diff
@@ -95,7 +95,7 @@ Scala
 Java
 :  @@snip [BasicPersistentBehaviorTest.java](/akka-persistence-typed/src/test/java/jdocs/akka/persistence/typed/BasicPersistentBehaviorTest.java) { #retentionCriteria #snapshottingPredicate }
 
-Snapshot deletion is triggered after saving a new snapshot.
+Snapshot deletion is triggered after successfully saving a new snapshot. Is saving a snaphot failed, snapshot deletion is not triggered.
```

Comment:
```suggestion
Snapshot deletion is triggered after successfully saving a new snapshot. If saving a snapshot failed, snapshot deletion is not triggered.
```

Reply:
I think it should says: "If saving a snapshot fails, the snapshot deletion is not triggered"